services/GetPriority.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.android_homework_1.services
+
+
+private val priorityMap: Map<String, Int> = mapOf(
+    "=" to 0,
+    "+" to 1,
+    "-" to 1,
+    "*" to 2,
+    "/" to 2
+)
+
+fun getPriority(operations: MutableList<String>): Int {

```

Comment:
для передачи параметров используем только обычный List
```suggestion
fun getPriority(operations: List<String>): Int {

```